spec/models/best_bet_spec.rb

```diff
@@ -140,6 +163,16 @@
       end
     end
 
+    describe "#author" do
+      it "should return the name of user" do
+        user_name = "Ahsoka Tano"
```

Comment:
![Ahsoka Tano poised for battle](https://media3.giphy.com/media/94NBPmEh41PJov4DTQ/giphy.gif)